rotkehlchen/tests/utils/dataimport.py

```diff
@@ -1641,6 +1705,102 @@ def assert_binance_import_results(rotki: Rotkehlchen):
             notes='Imported from binance CSV file. Binance operation: POS savings interest',
         ), HistoryEvent(
             identifier=6,
+            event_identifier='BNC_ac8a1a8ccd32547766f312428a59ac8ea667f72c33c0c4695b3fc82a7896645f',
+            sequence_index=0,
+            timestamp=ts_sec_to_ms(Timestamp(1640910825)),
+            location=Location.BINANCE,
+            asset=A_USDT,
+            balance=Balance(amount=FVal(500.00000000)),
+            event_type=HistoryEventType.TRANSFER,
+            event_subtype=HistoryEventSubType.NONE,
+            location_label='CSV import',
+            notes='Transfer Between Main and Funding Wallet',
+        ), HistoryEvent(
+            identifier=7,
+            event_identifier='BNC_ac8a1a8ccd32547766f312428a59ac8ea667f72c33c0c4695b3fc82a7896645f',
+            sequence_index=1,
+            timestamp=ts_sec_to_ms(Timestamp(1640910825)),
+            location=Location.BINANCE,
+            asset=A_USDT,
+            balance=Balance(amount=FVal(500.00000000)),
+            event_type=HistoryEventType.TRANSFER,
+            event_subtype=HistoryEventSubType.NONE,
+            location_label='CSV import',
+            notes='Transfer Between Main and Funding Wallet',
+        ), HistoryEvent(
+            identifier=8,
+            event_identifier='BNC_29a2d52a91b9bbb7213d4710ad190d587c4febc197373291289d85ae275ec7a5',
```

Comment:
Same for these 2 events `BNC_29a2d52a91b9bbb7213d4710ad190d587c4febc197373291289d85ae275ec7a5`